master_preset.xml

```diff
@@ -5754,6 +5755,171 @@
                     <check key="railway" value_on="halt" text="Railway halt" disable_off="true"/>
                 </checkgroup>
             </item> <!-- Station -->
+
+            <separator/> <!-- *** Road-based modes *** -->
+            <item name="Bus stop" type="node,way,closedway,multipolygon" icon="transport_platform_32003.png" preset_name_label="true">
+                <link wiki="Proposed_features/Public_Transport#Platform"/>
+                <space/>
+                <key key="public_transport" value="platform"/>
+                <checkgroup text="Transportation type" columns="3">
```

Comment:
If at all possible this checkgroup should be in a chunk to avoid code duplication.